src/api/cloud/project/dataprocessing/authorization/cloud-project-dataprocessing-authorization.iceberg.service.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * Interceptor purges serviceName from POST payload
+ * @type {{request(*): *}}
+ */
+const postInterceptor = {
+  request(req) {
+    req.data = null;
+    return req;
+  },
+};
+
+angular
+  .module('ovh-api-services')
+  .service('OvhApiCloudProjectDataProcessingAuthorizationIceberg', (iceberg) => iceberg('/cloud/project/:serviceName/dataProcessing/authorization', {
+    serviceName: '@serviceName',
+  }, {
+    post: {
+      method: 'POST',
+      url: '/cloud/project/:serviceName/dataProcessing/authorization',
+      interceptor: postInterceptor,
+      transformResponse: [],
```

Comment:
Is `transformResponse` still needed?

Reply:
Yes, we return an HTTP 204 (no content) but angular expects a json payload (as stated by default transformResponse). So here we specifically tell angular not to run any transformer on the response.

Reply:
Thanks for the explanation.